src/baldr/graphtile.cc

```diff
@@ -127,7 +130,8 @@ graph_tile_ptr GraphTile::Create(const std::string& tile_dir,
   }
 
   // Try to load a gzipped tile
-  std::ifstream gz_file(file_location + ".gz", std::ios::in | std::ios::binary | std::ios::ate);
+  std::ifstream gz_file(file_location.replace_extension(SUFFIX_COMPRESSED),
```

Comment:
hard to review these small differences. does file_location have `.gph` in it or not? was it previously doing `.gph.gz` and now its doing just `.gz`?

Reply:
yep it's `.gph.gz` now and was before too:
https://github.com/valhalla/valhalla/blob/9bf1d42cbd2fbbd1c4de19c0b0c0c364b2122182/valhalla/baldr/graphtile.h#L38